internal/serve/httphandler/receiver_handler_test.go

```diff
@@ -657,6 +660,7 @@ func Test_ReceiverHandler_GetReceivers_Success(t *testing.T) {
 						"failed_payments": "0",
 						"remaining_payments": "0",
 						"registered_wallets":"0",
+						"verifications": null,
```

Comment:
Can we return an empty array instead of `null`?

Reply:
👋 👀  having the verifications field set to `null` or `[]` here is misleading because you'd be essentially telling the API client that these receivers have no verification configured for them.

What you'd want to do is omit this field from the json response in these cases. I can think of two approaches to do that:

1. using `omitempty`, like we do at a few places like:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/fde25516e79b2c9cb44c78b32e3759c398b201ba/internal/anchorplatform/platform_api_service.go#L31

2. Creating a new response type for `GET /receivers`, that's almost the same as the one used in `GET /receiver/{id}` expect for the verifications field that is not provided.